README.md

```diff
@@ -38,9 +44,12 @@ The Street Work'App is designed to connect street workout enthusiasts by recomme
 
 3. **Park Events:**
     - Users can create and join events at parks to meet others and explore new workout locations.
-
-4. **Account Recommendations:**
-    - The app suggests users who frequent the same or nearby parks, helping to create workout communities. Users are also encouraged to schedule joint sessions.
+   
+4. **Progression**
+    - The app includes an achievement and progression system where users can track their milestones, unlock badges for completing specific goals. This gamification feature motivates users by rewarding consistent effort and providing a visual representation of their progress over time.
```

Comment:
Maybe adding an and would be more readable
users can track their milestones **and** unlock badges for completing specific goals